kythe/cxx/indexer/cxx/IndexerPPCallbacks.cc

```diff
@@ -354,11 +354,13 @@ void IndexerPPCallbacks::HandleKytheInlineMetadataPragma(
   clang::FileID pragma_file_id =
       Observer.getSourceManager()->getFileID(first_token.getLocation());
   if (pragma_file_id.isInvalid()) {
+    LOG(WARNING) << "Invalid file ID for kythe_inline_metadata";
```

Comment:
(optional) Does the token have a useful string rep we could include in the log?

Reply:
(argh the GitHub UI partially updated itself and I didn't see this until I committed)

The token's part of the pragma, so not really. This is a supremely strange case anyhow, because it indicates that we were unable to get the file ID for the file that we're *currently parsing*.

Reply:
Ah, yeah, that makes sense.